<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRUD_PBX</class>
 <widget class="QDialog" name="CRUD_PBX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRUD_PBX</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>291</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Fecha Llamada</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>268</width>
      <height>332</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditAnoI"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEditAnoL"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>AÃ±o</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEditMesL"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditMesI"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Dia</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLineEdit" name="lineEditDiaL"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEditHoraI"/>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QLineEdit" name="lineEditHoraL"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Hora</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Minuto</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="lineEditMinutoI"/>
     </item>
     <item row="10" column="3">
      <widget class="QLineEdit" name="lineEditMinutoL"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Segundo</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QLineEdit" name="lineEditSegundoL"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="lineEditSegundoI"/>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditDiaI"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Nombre del PBX : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNombrePBX"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Datos Llamada</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>341</width>
      <height>286</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditDuracionI">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEditDuracionL"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Duracion</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Origen</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditOrigenI"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEditOrigenL"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Destino</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="lineEditDestinoL"/>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditDestinoI"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Codigo de Cuenta</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEditCuentaI"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>Largo</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLineEdit" name="lineEditCuentaL"/>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>Longitud telefono interno:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="lineEditInternoL"/>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>Longitud de la trama</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLineEdit" name="lineEditTramaL"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButtonAceptar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
